### PR TITLE
Implemented frontend users

### PR DESCRIPTION
**Pull request by blacksph3re**

### CODE REVIEW

File: public/assets/js/angular-controller.js [hunk 1]
```diff
@@ -159,122 +159,27 @@ omsApp.controller('sidebarController', function($scope, $rootScope, $state) {
 });
 
 
-
```
Comment (i3anaan) [inline public/assets/js/angular-controller.js:162]: Could you summarize what you deleted here?

**Review by i3anaan — APPROVED** [review]
Hmm I have mixed feelings.
Aside from the clarification I would like to have on what you deleted (This is more curiosity that critique) I am not sure how strict we should be on these frontend merges. For example your country does not show up yet in your profile, is that a reason to not merge? It does not break anything else, it seems to work.

So I let it up to you, @blacksph3re, to decide if you would rather have a fully working merge, or do it more incrementally, seeing this missing country thing as a bug.

File: public/assets/js/angular-controller.js (hunk 1 above)
Comment (blacksph3re) [reply public/assets/js/angular-controller.js:162]: The example code from the template, that you see when you go to the template preview.

**Comment by blacksph3re** [comment]
Oh yeah the country issue is known to me, the backend does currently not merge the country into the address and I was hoping for a fix from the backend for this. But I can provide a workaround until then

**Comment by i3anaan** [comment]
If it is up to the backend then I suggest you merge this and make a JIRA issue on CORE :)
